src/main/java/opendcs/dao/ComputationDAO.java

```diff
@@ -419,51 +452,60 @@ public boolean checkCachedObjectOK(CachableDbObject ob)
 			return false;
 		}
 	}
-	
+
 	@Override
 	public DbComputation getComputationByName(String name)
 		throws DbIoException, NoSuchObjectException
 	{
-		String q = "select " + compTableColumns 
+		String q = "select " + compTableColumns
 			+ " from CP_COMPUTATION where COMPUTATION_NAME = '"
 			+ name + "'";
-		
-		try
-		{
-			ResultSet rs = doQuery(q);
-			if (rs.next())
+		Connection conn = db.getConnection();
+		try(
+			PreparedStatement getComp = conn.prepareStatement(
+				"select " + compTableColumns + " from CP_COMPUATION where COMPUTATION_NAME = ?"
```

Comment:
Type: table name CP_COMPUTATION is mispelled. This is causing some programs to fail.